Source/CronosPlayUnreal/Private/DefiWalletCoreActor.cpp

```diff
@@ -646,6 +648,70 @@ void ADefiWalletCoreActor::SendEthAmount(int32 walletIndex, FString fromaddress,
   }
 }
 
+void ADefiWalletCoreActor::SendEthAmountAsync(
+    int32 walletIndex, FString fromaddress, FString toaddress, FString amount,
```

Comment:
are the units or format (I guess decimal "0.00" or is it hex?) for the `amount` documented in the header?

Reply:
amount: ethdecimal
gasprice: wei
added "unit" to argument names